Mathlib/Algebra/Polynomial/Bivariate.lean

```diff
@@ -41,15 +41,88 @@ abbrev CC (r : R) : R[X][Y] := C (C r)
 lemma evalEval_C (x y : R) (p : R[X]) : (C p).evalEval x y = p.eval x := by
   rw [evalEval, eval_C]
 
+lemma evalEval_CC (x y : R) (p : R) : (CC p).evalEval x y = p := by
+  rw [evalEval_C, eval_C]
+
+lemma evalEval_zero (x y : R) : (0 : R[X][Y]).evalEval x y = 0 := by
+  simp only [evalEval, eval_zero]
+
+lemma evalEval_one (x y : R) : (1 : R[X][Y]).evalEval x y = 1 := by
```

Comment:
I am surprised that most of the lemmas here are not `@[simp]`!

Reply:
I guess they should be...